code/model/modelrender.cpp

```diff
@@ -2539,13 +2539,13 @@ void model_render_debug(int model_num, matrix *orient, vec3d * pos, uint flags,
 	gr_zbuffer_set(save_gr_zbuffering_mode);
 }
 
-void model_render_immediate(model_render_params *render_info, int model_num, matrix *orient, vec3d * pos, int render, bool sort)
+void model_render_immediate(model_render_params* render_info, int model_num, matrix* orient, vec3d* pos, int render, bool sort, vec3d show_ship_pos, bool is_using_showship)
```

Comment:
The vector should either be `const` reference or even a pointer that defaults to `nullptr` which would also remove the need for the additional `bool` parameter.